### PR TITLE
🌟 #1539 Refactor workspace users manager frontend

### PR DESCRIPTION
**Pull request by stephanevieira75**
https://user-images.githubusercontent.com/36481167/132562208-42c762b0-d88a-477b-b524-2aa18fe5eb84.mp4

### CODE REVIEW

File: twake/backend/node/src/services/workspaces/web/routes.ts [hunk 1]
```diff
@@ -89,10 +89,10 @@ const routes: FastifyPluginCallback<{
       if (!workspace) {
         throw fastify.httpErrors.notFound(`Workspace ${workspaceId} not found`);
       } else {
-        throw fastify.httpErrors.forbidden("Only workspace admin can perform this action");
+        throw fastify.httpErrors.forbidden("You must be member of the workspace");
       }
-    } else if (workspaceUser.role !== "admin") {
-      throw fastify.httpErrors.forbidden("Only workspace admin can perform this action");
+    } else if (workspaceUser.role !== "moderator") {
```
Comment (RomaricMourgues) [inline twake/backend/node/src/services/workspaces/web/routes.ts:94]: Could we replace this by the utils function "hasWorkspaceAdminLevel()" ?

File: twake/backend/node/src/services/workspaces/web/controllers/workspaces.ts [hunk 2]
```diff
@@ -191,7 +191,7 @@ export class WorkspacesCrudController
 
     const workspaceUserRole = await this.getWorkspaceUserRole(request.params.id, context);
 
-    if (workspaceUserRole !== "admin") {
+    if (workspaceUserRole !== "moderator") {
```
Comment (RomaricMourgues) [inline twake/backend/node/src/services/workspaces/web/controllers/workspaces.ts:194]: Could we replace this by the utils function "hasWorkspaceAdminLevel()" ?

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]
Also it seems we are missing the frontend side to switch from "admin" to "moderator".
+ I asked Roman to make sure every company admin is workspace moderator, I don't see this code anywhere yet

File: twake/backend/node/src/services/workspaces/web/controllers/workspaces.ts (hunk 2 above)
Comment (stephanevieira75) [reply twake/backend/node/src/services/workspaces/web/controllers/workspaces.ts]: done

File: twake/backend/node/src/services/workspaces/web/routes.ts (hunk 1 above)
Comment (stephanevieira75) [reply twake/backend/node/src/services/workspaces/web/routes.ts]: done

**Review by RomaricMourgues — APPROVED** [review]

**Pull request merged**